plugin/trino-hive-hadoop2/bin/run_hive_abfs_access_key_tests.sh

```diff
@@ -4,6 +4,10 @@ set -euo pipefail -x
 
 . "${BASH_SOURCE%/*}/common.sh"
 
+if ! check_gib_impacted trino-hive-hadoop2; then
```

Comment:
Actually, no need to pass the module, since all of this code is local to a single module. So just hard code it for this single module, and it simplifies to
```bash
abort_if_not_gib_impacted
```
I don't like to have the same boilerplate copied around a bunch of places.

Reply:
Looks much better, thanks!